code/defines/procs/announce.dm

```diff
@@ -116,13 +116,13 @@ datum/announcement/proc/Log(message as text, message_title as text)
 	command_announcement.Announce("It has come to our attention that an ion storm has formed.  Please monitor all electronic equipment for malfunctions.", "Anomaly Alert", new_sound = 'sound/AI/ionstorm.ogg')
 
 /proc/AnnounceArrival(var/mob/living/character, var/rank, var/join_message)
-	if (join_message && SSticker.current_state == GAME_STATE_PLAYING && SSjob.ShouldCreateRecords(rank))
-		if(issilicon(character))
-			global_announcer.autosay("A new [rank] [join_message].", ANNOUNSER_NAME)
-		else
+    if (join_message && SSticker.current_state == GAME_STATE_PLAYING && SSjob.ShouldCreateRecords(rank))
```

Comment:
Please use spaces not tabs

Reply:
You wrote that code and told me to use it!